.bazelrc

```diff
@@ -3,16 +3,34 @@ try-import submodules/dev-tools/.bazelrc
 # Specific Bazel build/test options.
 build --enable_platform_specific_config
 
-build:macos --cxxopt='-std=c++17'
-
-build:linux --cxxopt='-std=c++17'
-
-build:windows --cxxopt="/std:c++17"
+# Verbose failure logs when something goes wrong.
+build --verbose_failures
 
+# Use Clang as a default compiler on Windows.
 build:windows --compiler="clang-cl"
 
+# Compiler options.
+build:gcc --action_env=CC=gcc
 build:clang --action_env=CC=clang
 
+# By default, build eventuals in C++17 mode.
```

Comment:
I'd remove the "by default" bit (it doesn't add much: this whole file is defining defaults)